### DISCUSSION TITLE
stronghold bloqueado

### DISCUSSION THREAD

**Discussion by Osomillonario10** [iotaledger/firefly#8203] [Help & Support]
hola amigos IOTA , tengo el stronghold bloqueado ,que debo hacer para desbloquearlo gracias 
me pueden escribir la respuesta a mi correo : fabianmindira@gmail.com

gracias por su ayuda

**Comment by pcarreteros** [comment]
Hola. Has conseguido arreglar este asunto. Yo tmpc puedo acceder.

**Comment by Osomillonario10** [comment]
No me han dado solución

**Comment by begonaalvarezd** [comment]
Si no recordais la contraseña de vuestro stronghold, podeis importar el perfil con vuestra mnemonic de 24 palabras

**Comment by pcarreteros** [comment]
Hola Begoña, Gracias por contestar. No tengo la la mnemonic de 24 palabras.
No es que no recuerde la contraseña, estoy seguro que la contraseña que
estoy poniendo es la correcta.

Tengo los ficheros stronghold, el kdbx y la dirección del wallet, se te
ocurre alguna manera de recuperar las iotas?

Abrazo y gracias,
Pablo Carretero.

**Reply by begonaalvarezd** [reply]
La unica manera de recuperar tus IOTAs es teniendo tus claves privadas, que son o bien tus 24 palabras, el stronghold desbloqueado (se desbloquea con la contraseña correcta), o tu ledger en caso de que tengas tambien una ledger wallet.
En cuanto al `kdbx` es un archivo antiguo que imagino lo usaste cuando migraste, si ya migraste tus IOTAs, no lo necesitas más

Si el stronghold que tienes no eres capaz de desbloquearlo, puedes intentar buscar otro stronghold que tengas en el path de instalacion de firefly, puedes buscar por tus `.stronghold` y probar a ver si ese puede desbloquearse con la contraseña que recuerdas
- Windows: `%APPDATA%\Roaming\Firefly\__storage__/[profileID]`
- MacOS: `$HOME/Library/Application\ Support/Firefly/__storage__/[profileID]`
- Linux: `~/.config/Firefly/__storage__/[profileID]`
donde `[profileID]` es una carpeta con un ID alfanumerico

Mucha suerte 🤞🏼

**Comment by pcarreteros** [comment]
Pues es que me temo que para colmo de todos los colmos actualicé la versión
y se han borrado.... Sigo intentándolo.

Lo que tengo el migration.log. De ahí no puedo sacar la información, no?

Abrazo y muchas gracias.
Pablo Carretero.

**Reply by begonaalvarezd** [reply]
Disculpa, no entiendo muy bien el proceso por el cual has obtenido un stronghold, ha sido migrando? Si es así, puedes buscar otro stronghold en el path de instalacion de la legacy migration tool, puedes usar los mismos paths que te puse arriba pero en vez de Firefly busca en la carpeta `IOTA Legacy Migration Tool` en plan eg macOS `%APPDATA%\Roaming\IOTA Legacy Migration Tool\__storage__/[profileID]`

Para casos de servicio te recomiendo que uses nuestro discord y no github, puedes unirte en https://discord.iota.org/ y usar los canales `help` o `firefly-support`.
